Mathlib/CategoryTheory/DinatTrans.lean

```diff
@@ -0,0 +1,67 @@
+/-
+Copyright (c) 2023 Andrea Laretto. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrea Laretto
```

Comment:
```suggestion
Authors: Andrea Laretto, Fernando Chu
```

Reply:
Seems fair, I have put some effort now even though it's not in this PR yet :stuck_out_tongue: